src/Microsoft.Dnx.Runtime/Project.cs

```diff
@@ -233,12 +236,25 @@ internal static Project GetProjectFromStream(Stream stream, string projectName,
 
             project.Dependencies = new List<LibraryDependency>();
 
+            // Files to be packed along with the project
+            var packInclude = rawProject.ValueAsJsonObject(PackIncludePropertyName);
```

Comment:
> Perhaps with the ProjectReader change we could make the amount of data loaded configurable and then only have dnu pack load this much but that seems like a larger change.

I'll look into that. I was thinking of a runtime project but this might be a better option.